jena-fuseki2/jena-fuseki-ui/src/App.vue

```diff
@@ -29,11 +29,13 @@
 
 <script>
 import Menu from '@/components/Menu.vue'
+import Toast from '@/components/Toast.vue'
 
 export default {
   name: 'App',
   components: {
-    Menu
+    Menu,
+    Toast
```

Comment:
We used it in the template, but did not import it. Vue/Vite were happy with that, but the IDE warns you when you don't do it (depending on your project setup, that may raise errors)